modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPRequest.testcase

```diff
@@ -144,6 +144,74 @@ definition {
 		}
 	}
 
+	@description = "LPS-180356 - Verify that  the 'phone number' field is enabled  if it's not registered and it is possible to fill this"
+	@priority = 3
+	test IsPossibleFillPhoneNumberField {
+		task ("Given the Employee user don't have a phone number registered") {
+			ApplicationsMenu.gotoPortlet(
+				category = "User",
+				panel = "Control Panel",
+				portlet = "Users and Organizations");
+
+			EVPUtils.gotoContactInformation(
+				contactUser = "Contact",
+				enterUser = "Employee User",
+				inforUser = "Contact Information");
+
+			EVPUtils.assertContactInformation(inforMessage = "This user does not have any phone numbers");
+		}
+
+		task ("When he goes to Create Request Form") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "employee@liferay.com",
+				userLoginFullName = "Employee User");
+
+			EVPUtils.gotoEVP();
+
+			EVPRequest.goToCreateRequest();
+		}
+
+		task ("Then the 'phone number' field is enabled and He fill this") {
+			EVPUtils.assertTypeNumber(
+				assertNumber = 1234567890,
+				key_number = 1234567890);
+		}
+
+		task ("Given the Employee user have a phone number registered") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "test@liferay.com",
+				userLoginFullName = "Test Test ");
+
+			ApplicationsMenu.gotoPortlet(
+				category = "User",
+				panel = "Control Panel",
+				portlet = "Users and Organizations");
+
+			EVPUtils.gotoContactInformation(
+				contactUser = "Contact",
+				enterUser = "Employee User",
+				inforUser = "Contact Information");
```

Comment:
You can remove the macro: `ApplicationsMenu.gotoPortlet` since after the previous change I requested this will be inside the macro:`EVPUtils.gotoContactInformation`